pom.xml

```diff
@@ -184,7 +191,7 @@
       <dependency>
         <groupId>com.thoughtworks.xstream</groupId>
         <artifactId>xstream</artifactId>
-        <version>${xstream.version}</version>
+        <version>1.4.19</version>
```

Comment:
# <img title="Critical" alt="Critical" src="https://cdn.sonatype.com/iq-for-scm/1.0/red-bar.png" width="20" height="20"/> <img title="Direct Dependency" alt="Direct Dependency" src="https://cdn.sonatype.com/iq-for-scm/1.0/d-logo.png" width="20" height="20"/>  Sonatype IQ found critical issues introduced by com.thoughtworks.xstream : xstream : 1.4.19<br /><img title="Preview" alt="Preview" src="https://cdn.sonatype.com/iq-for-scm/1.0/Preview.svg" width="70" height="20"/>
Threat Level: <strong>Critical (9)</strong> \| [View Component Details in Sonatype Lifecycle](http://127.0.0.1:8070/ui/links/application/prcommentingappid/report/932fcfaf4ed84257ba5e264e210f5b03/componentDetails/e0e581d812aa92ae12f0?source=pr-line-commenting&utm_source=github)

## :shield: Recommendation
**Bumping to version 1.4.20** will resolve all policy violations for this component (as of _May 08, 2024_)
There are multiple **breaking changes**. This version upgrade may require significant effort.

## :page_facing_up: Security Issue Details

<details>
  <summary title="View all (2)">
    View all (2)
  </summary>
  <p></p>

| **Severity** | **Issue** | **Organization Policy Violation** |
| --- | --- | --- |
| <b>Threat level:</b> 9<br><b>CVSS Score:</b> 7.5<br><br><img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /> | [CVE-2022-41966] XStream serializes Java objects to XML and back again. Versions prior to 1.4.20 may allow a remote attacker to terminate the application with a stack overflow error, resulting in a denial of service only via manipulation the processed input stream. The attack uses the hash code implementation for collections and maps to force recursive hash calculation causing a stack overflow. This issue is patched in version 1.4.20 which handles the stack overflow and raises an InputManipulationException instead. A potential workaround for users who only use HashMap or HashSet and whose XML refers these only as default map or set, is to change the default implementation of java.util.Map and java.util per the code example in the referenced advisory. However, this implies that your application does not care about the implementation of the map and all elements are comparable. | [View Details](http://127.0.0.1:8070/ui/links/policyViolationReport/2d878c57dad84653bab248c48d9616af?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 9<br><b>CVSS Score:</b> 7.5<br><br><img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /> | [CVE-2022-40151] Those using Xstream to seralize XML data may be vulnerable to Denial of Service attacks (DOS). If the parser is running on user supplied input, an attacker may supply content that causes the parser to crash by stackoverflow. This effect may support a denial of service attack. | [View Details](http://127.0.0.1:8070/ui/links/policyViolationReport/4833ff3ec0f448c09b6dcc35a649ab40?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|

</details>